CodeEdit/Features/CEWorkspace/Views/EditCETaskView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  EditCETaskView.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 12/4/24.
+//
+
+import SwiftUI
+import Collections
+
+struct EditCETaskView: View {
+    @Environment(\.dismiss)
+    var dismiss
+
+    @Binding var task: CETask
+    @Binding var settings: CEWorkspaceSettingsData.TasksSettings
+
+    var body: some View {
+        VStack(spacing: 0) {
+            CETaskFormView(
+                task: $task
+            )
+            Spacer()
+            Divider()
+            HStack {
+                Button("Remove...") {
```

Comment:
```suggestion
                Button("Remove") {
```

Reply:
This is the design we agreed on, I think we should comment on this before making this change